llvm/lib/Target/AArch64/AArch64InstrInfo.td

```diff
@@ -10155,7 +10155,19 @@ let Predicates = [HasFP8] in {
 let Predicates = [HasFAMINMAX] in {
  defm FAMAX : SIMDThreeSameVectorFP<0b0, 0b1, 0b011, "famax", null_frag>;
  defm FAMIN : SIMDThreeSameVectorFP<0b1, 0b1, 0b011, "famin", null_frag>;
-} // End let Predicates = [HasFAMAXMIN]
+} // End let Predicates = [HasFAMINMAX]
+
+let Predicates = [HasNEON, HasFAMINMAX] in {
```

Comment:
You shouldn't need to break the block here but instead use the new `Predicates` definition for the instruction defs as well as your new isel patterns.

Reply:
Thanks, done.